packages/upswyng-web/src/utils/schedule.ts

```diff
@@ -72,3 +79,32 @@ export const orderSchedule = (schedule: TSchedule[]) => {
   }
   return schedule;
 };
+
+const getOpensAtText = ({ open }: TScheduleItemOpenClose) =>
+  moment(open).calendar(new Date(), {
+    sameDay: "[today at] h:mm A",
+    nextDay: "[tomorrow at] h:mm A",
+    nextWeek: "dddd [at] h:mm A",
+    sameElse: "MMM Do",
```

Comment:
whats a `sameElse`

Reply:
That's moment js's indication that the passed date is more than a week away from the comparison date. more in their [docs](https://momentjs.com/docs/#/displaying/calendar-time/)